integrationTests/testProcessorNode.go

```diff
@@ -723,3 +724,21 @@ func (tpn *TestProcessorNode) syncMetaNode(nonce uint64) error {
 
 	return nil
 }
+
+// SetAccountNonce sets the account nonce with journal
+func (tpn *TestProcessorNode) SetAccountNonce(nonce uint64) error {
+	nodePubKeyBytes, _ := tpn.SkTxSign.GeneratePublic().ToByteArray()
```

Comment:
You already have this computed as tpn.PkTxSignBytes

Reply:
Changed.